test/inductor/test_torchinductor_dynamic_shapes.py

```diff
@@ -60,7 +60,7 @@
         ("cuda"), is_skip=True
     )
     test_failures["test_batch_norm_2d_dynamic_shapes"] = TestFailure(
-        ("cuda"), is_skip=True
+      "cuda", is_skip=True
```

Comment:
Not sure why the stack missed this

Reply:
I see the previous lines weren't changed. Maybe I had done this manually for some reason; I can't remember.

I'm gonna also change the previous lines.